os/drivers/testcase/kernel_testcase_drv.c

```diff
@@ -169,6 +160,44 @@ static int kernel_test_drv_ioctl(FAR struct file *filep, int cmd, unsigned long
 	}
 	break;
 
+	case TESTIOC_CLOCK_ABSTIME2TICKS_TEST: {
```

Comment:
I don't understand why you modify it, If you made this because of clock_abstime2ticks to increase coverage, timer_settime already calls it and it is registered in syscall.csv.

Reply:
@Taejun-Kwon 
I modified it again. This was only test this function.